docs/configuration/device-config/lora.mdx

```diff
@@ -11,6 +11,10 @@ import LoRaRegions from "../../blocks/_lora-regions.mdx";
 
 The LoRa config options are: Region, Modem Preset, Max Hops, Transmit Power, Bandwidth, Spread Factor, Coding Rate, Frequency Offset, Transmit Enabled, Channel Number and Ignore Incoming Array. LoRa config uses an admin message sending a `Config.LoRa` protobuf.
 
+:::note
+In order to communicate fully, devices within a mesh must have identical settings for Region and Modem Preset, or identical custom Modem settings.
```

Comment:
Great admonition, it's something I want to convey visually as well down the line.